projects/dsp-ui/src/lib/viewer/views/list-view/resource-list/resource-list.component.html

```diff
@@ -1,26 +1,36 @@
+<!-- When withMultipleSelection is false and user can select only one resource at a time to view -->
 <mat-list class="resource-list">
-    <mat-list-item
+    <div
         class="link"
-        [class.selected-resource]="selectedResourceIdx === i"
+        [class.selected-resource]="selectedResourceIdx.indexOf(i) > -1"
         [class.border-bottom]="!last"
-        *ngFor="let res of resources.resources; let i = index; let last = last"
-        (click)="resourceSelected.emit(res.id)">
-        <mat-icon matListIcon>image_search</mat-icon>
-        <p matLine class="res-class-label">
-            {{ res.entityInfo.classes[res.type].label }}
-        </p>
-        <h3 matLine class="res-class-value">{{ res.label }}</h3>
+        *ngFor="let resource of resources.resources; let i = index; let last = last">
+        <mat-list-item>
+            <mat-icon matListIcon>image_search</mat-icon>
+            <div matLine (click)="viewResource({ checked: true, resListIndex: i, resId: resource.id, isCheckbox: false})">
+                <p matLine class="res-class-label">
+                    {{ resource.entityInfo.classes[resource.type].label }}
+                </p>
+                <h3 matLine class="res-class-value">{{ resource.label }}</h3>
```

Comment:
Could you set the style for `res-class-label` and `res-class-value` to `margin: 0;`? Otherwise it needs too much space.

Reply:
It is the case only on big screens. `margin:0` removes all the space and it looks crowded. I will make it 'margin: 5px 0', if it ok?

Reply:
It's also on small screen to height:
![Screen Shot 2021-07-07 at 12 52 45](https://user-images.githubusercontent.com/4253580/124747645-6fa68c00-df22-11eb-8068-528b966fc7ec.png)

And this is how it looks with `margin: 0`:
![Screen Shot 2021-07-07 at 12 56 34](https://user-images.githubusercontent.com/4253580/124747953-ce6c0580-df22-11eb-9041-3175bfae6cfc.png)

If you want to add a margin, we should follow to rule of 8. In this case you could set it to `margin: 4px 0;`

Reply:
I looks like this on my machine with current style.

![image](https://user-images.githubusercontent.com/25031092/124749213-3cfd9300-df24-11eb-8aa4-ad7236b9ccaf.png)

Anyways I will update the margin.

Reply:
Hm... that's weird 😳 it looks different on the two machines. But why?

Reply:
Are we using same version? I took this screenshot from `https://admin.test.dasch.swiss/` server.  I did not run the DSP-APP locally to check it there.

For `dsp-ui-lib` server running locally, it looks like below. It does show some spacing. So I assumed that in DSP-APP there are some extra styles applied for this as I did not change this style in this PR until now.

![image](https://user-images.githubusercontent.com/25031092/124754143-28bc9480-df2a-11eb-945c-2b6f2c82eaa0.png)

Reply:
I added the DSP-UI lib from your branch to DSP-APP locally (with yalc) and was running your implementation there. The version on  `https://admin.test.dasch.swiss/` doesn't have your implementation. Of course it looks different 😉

Reply:
In your screenshot you have the same big spaces that I have in DSP-APP. This is why I wrote to set the margin to 0 pixels.